Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorSmoothingEffect.py

```diff
@@ -342,8 +342,12 @@ def smoothMultipleSegments(self):
       smoothedBinaryLabelMap.ShallowCopy(stencil.GetOutput())
       smoothedBinaryLabelMap.SetImageToWorldMatrix(imageToWorldMatrix)
       # Write results to segments directly, bypassing masking
-      slicer.vtkSlicerSegmentationsModuleLogic.SetBinaryLabelmapToSegment(smoothedBinaryLabelMap,
-        segmentationNode, segmentId, slicer.vtkSlicerSegmentationsModuleLogic.MODE_REPLACE, smoothedBinaryLabelMap.GetExtent())
+      oldOverwriteMode = self.scriptedEffect.parameterSetNode().GetOverwriteMode()
+      self.scriptedEffect.parameterSetNode().SetOverwriteMode(slicer.vtkMRMLSegmentEditorNode.OverwriteAllSegments)
```

Comment:
Why this is needed?

Reply:
I investigated more, and the other overwrite mode changes in the AutoComplete effect were not required (after some bug fixing).

I've changed the overwrite mode to OverwriteVisibleSegments. I'm not sure if I understand how the layer mode would differ from the overwrite mode.